lib/sejfguru/assets/assets.ex

```diff
@@ -166,4 +168,13 @@ defmodule Sejfguru.Assets do
   def change_asset(%Asset{} = asset) do
     Asset.changeset(asset, %{})
   end
+
+  defp location_condition(query, :all_locations), do: query
+  defp location_condition(query, locations) do
+    Ecto.Query.where(query, [a], a.location_name in ^locations)
+  end
+
+  defp chosen_location(location) do
+    Location.locations().cities[location] || Location.locations().other[location]
```

Comment:
this could be a function in `Assets.Location` module and it could also return some default if the value given is not found in the map - to prevent the error if user gives an invalid `location` param
and I think better naming would be something like `get_location_details(location)`, what do you say?